stacks-signer/src/signerdb.rs

```diff
@@ -134,15 +175,29 @@ impl SignerDb {
         hash: &Sha512Trunc256Sum,
     ) -> Result<(), DBError> {
         debug!("Deleting block_info: sighash = {hash}");
+
         self.db.execute(
             "DELETE FROM blocks WHERE reward_cycle = ? AND signer_signature_hash = ?",
-            &[reward_cycle.to_string(), format!("{}", hash)],
+            &[
+                &u64_to_sql(reward_cycle)? as &dyn ToSql,
+                &format!("{}", hash),
```

Comment:
`hash.to_string()`

Reply:
This is unrelated to the current PR. Changing this to `.to_string()` would render this function useless since the hashes are inserted using `format!("{}", hash` in the `insert_block` function.

Reply:
So change it there too?

Reply:
Done.